src/Forms/FormLabel.js

```diff
@@ -47,7 +49,9 @@ FormLabel.propTypes = {
     /** Set to **true** if child is InlineHelp component */
     isInlineHelp: PropTypes.bool,
     /** Set to **true** for required input fields */
-    required: PropTypes.bool
+    required: PropTypes.bool,
+    /** Set to **true** to add the `:` character at the end of a label as pseudo element.*/
+    withColon: PropTypes.bool
```

Comment:
Given a choice between `includeColon` or `withColon`, I like `includeColon` better. 

It matches the prop `includeEmptyOption`